Makefile

```diff
@@ -29,7 +29,8 @@ wasm:
 .PHONY: test
 test: PKGS ?= "./..."
 test: build
-	@TZ=UTC go test -ldflags="$(LDTESTFLAGS)" -timeout=30s $(PKGS)
+	@TZ=UTC go test -ldflags="$(LDTESTFLAGS)" -timeout=30s -covermode=atomic -coverprofile=cover.out -coverpkg=./... $(PKGS)
```

Comment:
Suggestion: Consider moving it to a separate target, or at least the last line. Running `go tool cover -func=cover.out` without some comparison/analysis does not give much hint.